Formula/libxo.rb

```diff
@@ -0,0 +1,33 @@
+class Libxo < Formula
+  desc "Allows an application to generate text, XML, JSON, and HTML output"
+  homepage "https://juniper.github.io/libxo/libxo-manual.html"
+  url "https://github.com/Juniper/libxo/releases/download/0.8.4/libxo-0.8.4.tar.gz"
+  sha256 "3b3a33b706075fdc478782fdc65db3e7f9167550f7d71b95e704236a797e7fd3"
+  head "https://github.com/Juniper/libxo"
+
+  option "without-test", "Skip build-time testing (not recommended)"
+
+  depends_on "autoconf" => :build
+  depends_on "libtool" => :build
+
+  def install
+    args = %W[--prefix=#{prefix}]
+
+    system "./configure", *args
+    system "make", "test" if build.with? "test"
```

Comment:
we don't allow this except for security software like openssl

Reply:
Sorry, just to be clear: you don't allow running tests or skipping tests?

Reply:
right

Reply:
we don't allow `make check` or `make test`